Makefile

```diff
@@ -192,13 +192,6 @@ _check_iasl:
 	@echo -n "Checking iasl..."
 	$(eval IASL_NAME := acpica-unix2-$(IASL_VER))
 	$(eval IASL_URL := "https://acpica.org/sites/acpica/files/$(IASL_NAME).tar.gz")
-ifneq ($(shell $(IASL) -v 2>/dev/null | grep $(IASL_VER)),)
-# iASL compiler is already available in the system.
-	@rm -rf $(IASL_DIR)
-	@echo "OK"
-else
```

Comment:
Hi Phil, this just remove the downloaded iasl when the matching version is available in the system. Can you show me how you met this issue ?

Reply:
I cannot duplicate it now, there must be something else I have touched other than makefile to cause this issue, please ignore this PR if you cannot duplicate either via 
making the platform "JadePkg" twice in the row, sorry about troubling you...

Reply:
No problem, just open a new one if you are able to reproduce it. Anyway, I find other changings still have their value.

Reply:
I think it might be caused by "edk2-platforms.config": by the time that I had not override this file path, no OEM section within the one in the "edk2-ampere-tools" either, so no "IASL_VER" could be parsed, which caused this issue. but debugging this issue made by myself is not priority task, just for your reference.

Reply:
If you want to change isal version, you need pass this variable in your make command

`# make IASL_VER=<iasl_version> ...`

Reply:
Thanks for the tip!
BTW, if modification other than the "IASL_VER" related can be merged, it would be very helpful for me to make this repository a submodule of my project.

Reply:
I can duplicate while the make target is "all", please help to confirm; I make BIOS like this:
  make -f $WORKSPACE/edk2-ampere-tools/Makefile \
      BOARD_SETTING=$WORKSPACE/srp-ac01-fw-aptv-bin-r1.06.20210528/altra_firmware_sdk/bin/board_settings/jade_board_setting_1.06.20210526.bin \
      ATF_SLIM=$WORKSPACE/srp-ac01-fw-aptv-bin-r1.06.20210528/altra_firmware_sdk/bin/atf/altra_atf_signed_1.06.20210526.slim \
      LINUXBOOT_BIN=$WORKSPACE/flashkernel \
      all